Cargo.toml

```diff
@@ -1,30 +1,77 @@
 [package]
-name = "sommelier_steward"
+name = "steward"
+authors = []
 version = "3.0.0"
-edition = "2021"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[workspace]
-default-members = [
-	"steward",
-	"steward_abi",
-	"steward_abi_build",
-	"steward_proto_rust",
-	"steward_proto_rust_build",
-]
-members = [
-	"steward",
-	"steward_abi",
-	"steward_abi_build",
-	"steward_proto_rust",
-	"steward_proto_rust_build",
-]
+edition = "2018"
```

Comment:
Why are we downgrading rust edition? Is this related to importing the orchestrator dependencies?